src/detect-dnp3.h

```diff
@@ -0,0 +1,58 @@
+/* Copyright (C) 2015 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+#ifndef __DETECT_DNP3_H__
+#define __DETECT_DNP3_H__
+
+/**
+ * Struct for mapping symbolic names to values.
+ */
+typedef struct DNP3Mapping_ {
```

Comment:
If inspect engines go into keyword file we can limit the scope of these data types to the .c file as well I think.